### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by tako-black**
Add period at the end of a sentence

### CODE REVIEW

File: docs/Style.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ All the attribute names and values are a subset of what works on the web. For la
...
-All the core components accept a style attribute
+All the core components accept a style attribute.
 
 ```javascript
 <Text style={styles.base} />
 <View style={styles.background} />
 ```
 
-and also accepts an array of styles
+And also accepts an array of styles.
```
Comment (sahrens) [inline docs/Style.md]: Can you make this `They also accept an array of styles.` instead?

Comment (tako-black) [inline docs/Style.md]: I can. I'll do that.

File: docs/Style.md [hunk 2]
```diff
@@ -37,14 +37,14 @@ All the attribute names and values are a subset of what works on the web. For la
...
-All the core components accept a style attribute
+All the core components accept a style attribute.
 
 ```javascript
 <Text style={styles.base} />
 <View style={styles.background} />
 ```
 
-and also accepts an array of styles
+They also accepts an array of styles.
```
Comment (sahrens) [inline docs/Style.md]: `accepts` should be singular - `They also accept an`

**Comment by vjeux** [comment]
Thanks!

**Pull request merged**